ADD  -- Task 4(I2): Add rol student

marvicmar opened the pull request.

marvicmar committed:
Add rol student

alepervaz reviewed (approved):
el código implementado esta completo y de forma correcta

alesanfe reviewed (changes requested):
Las restricciones están mal, el máximo está incluido.

marvicmar committed:
Update Student.java

Correction of the limit of the restictions

alesanfe committed:
fix(init): Student entity creation

Comments added to follow the format of the entities in the rest of the project.

alepervaz reviewed (approved):
todo correcto
